server/src/main/java/io/crate/planner/operators/PrintContext.java

```diff
@@ -65,6 +72,15 @@ public final PrintContext nest(List<Consumer<PrintContext>> children) {
         return this;
     }
 
+    PrintContext costs(LogicalPlan logicalPlan) {
+        if (planStats == null) {
+            return this;
+        }
+        var stats = planStats.get(logicalPlan);
+        text(" (rows:" + stats.numDocs() + ")");
+        return this;
+    }
+
```

Comment:
To my understanding `what` to print should be managed by `logicalPlans`, WDYT?

Reply:
Once the keyword(`explain` or `explain costs`) is decided I can have another look at the parser changes, LGTM otherwise.

Reply:
I thought the same in the beginning but the `PrintContext` solely exists to print `LogicalPlans` and the costs look the same for all `LogicalPlans` therefore it is a lot simpler to add the logic there.

Reply:
I moved the costs ouput now into the Logical Plan as you suggested.